docs/OV_Runtime_UG/multi_device.md

```diff
@@ -137,186 +115,46 @@ You can use the [configure devices](supported_plugins/config_properties.md) to q
-
-
-### Enumerating Available Devices
-The OpenVINO Runtime API features a dedicated methods to enumerate devices and their capabilities. See the [Hello Query Device Python Sample](../../samples/python/hello_query_device/README.md). This is example output from the sample (truncated to device names only):
+To see how the Multi-Device execution is used in practice and test its performance, take a look at OpenVINO™ samples. All samples supporting the `-d` command-line option (which stands for “device”) will accept the plugin out-of-the-box. The Benchmark Application will be a perfect place to start – it presents the optimal performance of the plugin without the need for additional settings, like the number of requests or CPU threads. 
+Here is an example command to evaluate performance of HDDL+GPU: 
 
 ```sh
-  ./hello_query_device
-  Available devices:
-      Device: CPU
-  ...
-      Device: GPU.0
-  ...
-      Device: GPU.1
-  ...
-      Device: HDDL
-```
-
-A simple programmatic way to enumerate the devices and use with the multi-device is as follows:
-
-@sphinxdirective
-
-.. tab:: Python
-
-    .. doxygensnippet:: docs/snippets/ov_multi.py
-       :language: python
-       :fragment: [available_devices_1]
-
-@endsphinxdirective
-
-Beyond the trivial "CPU", "GPU", "HDDL" and so on, when multiple instances of a device are available the names are more qualified. For example, this is how two Intel® Movidius™ Myriad™ X sticks are listed with the hello_query_sample:
-
-```bash
-  ...
-      Device: MYRIAD.1.2-ma2480
-  ...
-      Device: MYRIAD.1.4-ma2480
+./benchmark_app –d MULTI:HDDL,GPU –m <model> -i <input> -niter 1000
 ```
 
-So the explicit configuration to use both would be "MULTI:MYRIAD.1.2-ma2480,MYRIAD.1.4-ma2480". Accordingly, the code that loops over all available devices of "MYRIAD" type only is below:
+For more information, refer to the [C++](../../samples/cpp/benchmark_app/README.md) or [Python](../../tools/benchmark_tool/README.md) version instructions.	
 
 @sphinxdirective
+.. note::
 
-.. tab:: Python
-
-    .. doxygensnippet:: docs/snippets/ov_multi.py
-       :language: python
-       :fragment: [available_devices_2]
+   You can use the FP16 IR to work with auto-device.
```

Comment:
MULTI

Reply:
the message is that no need to use the FP32 IR when any device in the list doesn't suppurt the FP16, the conversion is automacally happens by the OV

Reply:
The whole point reworded, I hope now it's more clear